cmd/otelcol/config/collector/agent_config.yaml

```
@@ -156,8 +156,9 @@ exporters:
     endpoint: "${SPLUNK_GATEWAY_URL}:4317"
     tls:
       insecure: true
-  # Debug
-  debug:
+  # This component has been deprecated by the "debug" exporter,
+  # which will be adopted in a future release.
+  logging:
```

I think this should be reverted:
```suggestion
  debug:
```

---

Since `logging` was replaced by `debug` in #3755, the tests are expecting `debug` now.

---

I've requested we hold back on this change: https://github.com/signalfx/splunk-otel-collector/pull/3768#discussion_r1358264637